### DISCUSSION TITLE
Escape not handled in many dialogs on GitHub

### DISCUSSION THREAD

**Discussion by mskelton** [community/community#45549] [Accessibility]
### Select Topic Area

Product Feedback

### Body

I love much of the new updates to the GitHub experience especially the new code search.  However, there is a fairly serious usability issue in that you cannot press escape to close these new dialogs.  While you can tab to the close button, these new dialogs do not align well with other areas of GitHub and other sites that allow users to press escape to close dialogs.

<img width="1570" alt="image" src="https://user-images.githubusercontent.com/25914066/215134917-2b859e53-a472-4b18-a6e3-0ce0115ebb79.png">
<img width="477" alt="image" src="https://user-images.githubusercontent.com/25914066/215134804-51319c7e-ffb1-4b78-8a63-df29a3031d3f.png">

**Comment by cehfisher** [comment]
Hi @mskelton - we wanted to follow up on this issue. In reviewing this issue, it appears that the escape key now works as requested. Is this still an issue for you? If it is resolved, could you please close out this issue? If it is not resolved, what additional changes are needed? Please supply detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)

**Reply by mskelton** [reply]
@cehfisher Yep, it does work now, thanks for following up!
